Include/internal/pycore_gc.h

```diff
@@ -262,11 +262,11 @@ extern PyObject *_PyGC_GetReferrers(PyInterpreterState *interp, PyObject *objs);
 
 // Functions to clear types free lists
 extern void _PyGC_ClearAllFreeLists(PyInterpreterState *interp);
-extern void _Py_ClearFreeLists(_PyFreeListState *state, int is_finalization);
+extern void _PyObject_ClearFreeLists(_PyFreeListState *state, int is_finalization);
 extern void _PyTuple_ClearFreeList(_PyFreeListState *state, int is_finalization);
 extern void _PyFloat_ClearFreeList(_PyFreeListState *state, int is_finalization);
 extern void _PyList_ClearFreeList(_PyFreeListState *state, int is_finalization);
-extern void _PySlice_ClearCache(_PyFreeListState *state);
+extern void _PySlice_ClearCache(_PyFreeListState *state, int is_finalization);
 extern void _PyDict_ClearFreeList(_PyFreeListState *state, int is_finalization);
 extern void _PyAsyncGen_ClearFreeLists(_PyFreeListState *state, int is_finalization);
 extern void _PyContext_ClearFreeList(_PyFreeListState *state, int is_finalization);
```

Comment:
FWIW, I personally would put these function declarations in the respective type-specific header files, e.g. `_PyTuple_ClearFreeList()` in pycore_tuple.h.  That's the first place I'd think to look for a type-specific declaration like this.  The second place I'd look is pycore_freelist.h.  I would not think to look for it in pycore_gc.h.